release/webhook

```diff
@@ -114,6 +115,17 @@ class GithubHandler(http.server.BaseHTTPRequestHandler):
             self.fail('unsupported event ' + event)
             return
 
+        ref_type = request.get('ref_type', '')
+        if ref_type != 'tag':
+            self.fail('Ignoring ref_type %s, only doing releases on tags' % ref_type, reason=501)
```

Comment:
Did you mean `code=501` here, instead of `reason`?

Reply:
Argh! Indeed I did, well spotted! Pushed, and also updated/pushed release container.